### DISCUSSION TITLE
煽り・釣りタイトル記事とガイドラインについて

### DISCUSSION THREAD

**Discussion by radian-jp** [increments/qiita-discussions#490] [ヘルプ] [answered]
最近、このような記事が投稿されていました。

（指摘があったためリンク削除）

この記事に限ったことではないですが、注目を集めるために妙に煽ったり、目を引くようなタイトルを付けて実際の記事内容と食い違ったり、概要が把握できない記事を時々見かけます。コミュニティガイドラインの「読み手にとって適切な表現を心がけよう」には反していると感じます。

ガイドラインは規約のように明確に禁止されている訳ではないので、あまり強く注意するのも憚られるのですが、この手のタイトルのガイドライン違反を通報して修正されたのを見たことが無いので、実際の所、とりあえず注目集めればOKみたいな状態であることは否めないと思います。

そうなってくると、ガイドラインはあるだけで形骸化してるのでは？と思わざるを得ないのですが、運営としては、こういった行為は問題ないという認識でよろしいのでしょうか？明確に線引きするのが難しい問題だと理解は出来るのですが、お聞かせ願えるとありがたいです。

**Comment by JunichiIto** [comment]
リンク先の記事のタイトルはたしかに僕も違和感がありました。
当該記事は「注目を集めるために妙に煽ったり、目を引くようなタイトルを付けて実際の記事内容と食い違ったり、概要が把握できない」タイトルだと思います。
実際僕も「これ、何が書いてあるんだろう？」と思ってついクリックしてしまいましたし。

「明確に線引きするのが難しい問題」という点についても同じく同意ですが、こういうタイトルの記事ばかりが増えるとQiitaが「まとめサイト」みたいになってしまいそうです。

ちなみにご本人は「意図的にそうした」とのことです。
（リンク削除）
ですが、Qiitaというサービス・コミュニティをそういう形で利用するのは果たしていかがなものか（明確な意図や高尚な目的があるなら釣りタイトルは許されるのか？今後同じような事例が続いても許容するのか？）、と思ってしまいます。

**Reply by Verclene** [reply]
個人的な尺度であることを承知で敢えて書きますが,今回変にこの話が盛り上がってるのは一部閲覧者の持つ2chアレルギーのバイアスだと思います.個々の具体的な改善案に関してはここでは言及しませんが.
ガイドラインの厳格運用に関しては反対しませんが,Qiitaは良くも悪くも筆者が持つセンスと技術知識そしてコミュニティによって成立しますので,読んで面白い要素は少なからず要求されます.そういうのがいらないなら完全ナレッジオンリーのWikiでも見ていればいいと個人的には思います.少なくともQiitaをそういった方向にはもっていってほしくないです.

>守らなくても何もマイナスにならないなら、何のためのガイドラインか判りませんね。そういった手法が見過ごされ、まかり通るのは今後にいい影響があるとはとても思えません。ガイドライン守らなくていいなら、自分も釣りタイトルつけとくかって人が増えなければいいですね。

あとこれ「ガイドラインを守っていない」前提でお話しされている辺り無意識な悪意を感じます.今回は「ガイドラインに抵触するかどうか」の確認が必要な段階での対応なのであって,明らかに守らない人への対応はまた別段階になると思われます.

**Reply by radian-jp** [reply]
> 今回変にこの話が盛り上がってるのは一部閲覧者の持つ2chアレルギーのバイアスだと思います.

ここまでの話を全て読んでその感想であれば、特に何も申し上げる事はないです。
件の記事は判り易い例として取り上げさせて貰いましたが、議論したいのは別にそこじゃないです。面白い要素の有無と、ガイドラインを守ってるかどうかは別の話です。

**Comment by tanaka-asami415** [comment]
@radian-jp @JunichiIto 
ご意見ありがとうございます 🙏

コミュニティガイドライン
[コミュニティガイドライン違反の記事に対する運営の方針について](https://help.qiita.com/ja/articles/qiita-community-guideline#part-cf79556a40a47ce0)
にも記載している通りの回答となってしまい恐縮ですが

運営として、コミュニティガイドラインに違反しているものに対しては、
原則としてユーザーからの通報を元に当社で検討・判断をし、修正の依頼をする等の対応を行なっています。

ご記載いただいている通り「明確に線引きするのが難しい問題」であり、
我々も日々、悩みながらいただいた通報を確認しています。

運営の強権によって管理することは、過去の反省も含め、コミュニティ形成にプラスになることばかりでもないと考えています。
そのため、運営が良し悪しを判断するのではなく、
コミュニティの反応を皆様へお伝えして、気づきを与えることが運営の役割だと考えています。 

ご理解いただけますと幸いです。

また、【初めにお読みください】 #116 にも記載しておりますが
特定のアカウントを記載する行為はお控えいただけますと幸いです🙇

**Reply by radian-jp** [reply]
> また、【初めにお読みください】 https://github.com/increments/qiita-discussions/discussions/116 にも記載しておりますが
特定のアカウントを記載する行為はお控えいただけますと幸いです🙇

失礼しました、リンクを削除しました。

> 運営の強権によって管理することは、過去の反省も含め、コミュニティ形成にプラスになることばかりでもないと考えています。
そのため、運営が良し悪しを判断するのではなく、
コミュニティの反応を皆様へお伝えして、気づきを与えることが運営の役割だと考えています。

気付きを与えるとのことですが、良くない事と判ってやっている人に対して効果があるかというと、恐らくないでしょう。注目を集めた時点で、既に目的が達成されているからです。守らなくても何もマイナスにならないなら、何のためのガイドラインか判りませんね。そういった手法が見過ごされ、まかり通るのは今後にいい影響があるとはとても思えません。ガイドライン守らなくていいなら、自分も釣りタイトルつけとくかって人が増えなければいいですね。

他の事例としては、未経験から〇ヶ月でエンジニア系も、具体的なスクール名出しちゃってる場合もあるけど、ああいうのは宣伝になるんでしょうか？（宣伝は明確に規約で禁止の項目があった気がしますが）
スクールの提灯記事かなって思ってしまいます。

**Reply by JunichiIto** [reply]
@tanaka-asami415 
返信ありがとうございます。

すいません、リンクは削除しました。

最終判断は運営に委ねられるべきだと思うので、通報やこうした意見にどう対処するのかは運営のみなさんにお任せしています（いつもありがとうございます）。
なので、「あの記事は納得がいかないから削除してほしい」みたいなことは言うつもりはありません。

とはいえ、繰り返しになりますが、「線引きは難しい」というのは十分承知した上で、Qiitaというサービス・コミュニティ上で煽りや釣りを意図的にやる（コミュニティの規模や特性を理解した上で、明確な意思を持って自分が利する行動を取る）のは、コミュニティにとって好ましくないと考えています。
（ちなみに「そういうことをやりたいなら自分のブログでやれ、Qiitaでレバレッジを効かそうとするな」というのが僕の考えです）

何も声を上げなければ、「やったもん勝ち / 正直者が馬鹿を見る」ような状態になってしまうと思ったので、今回はあえて意見させてもらいました。

今後も同じような事例が続くようであれば、ガイドラインの見直しや運営からのアナウンス等を検討していただきたいです。
よろしくお願いいたします。

**Reply by JunichiIto** [reply]
> 何も声を上げなければ、「やったもん勝ち / 正直者が馬鹿を見る」ような状態になってしまうと思ったので、今回はあえて意見させてもらいました。

さらに付け加えるなら、「こうやればバズる」「これぐらいは許容範囲っぽい」という成功体験が投稿者や（一部の）読者に刷り込まれてしまわないか、という懸念もあります。
（そういう意味では早めにタイトルの修正はお願いしても良い気がしますが……）

**Comment by tanaka-asami415** [comment, marked as answer]
@radian-jp @JunichiIto 
リンクを削除いただきありがとうございます🙇

また日頃からコミュニティを支えてくださるだけでなく、このようにご意見をくださりありがとうございます🙏

あまり詳しく書いてしまうと悪用されてしまう可能性があるため
詳細をお伝えすることができないのですが、全てを一律に対応している訳ではなく
問題になる頻度や与える被害の大きさなどに合わせた対応を行っております。

今回も運営より連絡は行っており、今後の対応に関しましても、コミュニティガイドラインに沿って対応してまいりますので
ご理解いただけますと幸いです。

**Reply by radian-jp** [reply]
答えにくい質問だったと思われますが、返答頂きありがとうございます。
完全に納得出来た訳ではないですが、通報に対して個別に確認して対応されているという事で理解はしました。一利用者として危惧している所が伝わりましたら、と思います。
